app/utils/cps/kredits.js

```diff
@@ -0,0 +1,25 @@
+import { computed } from '@ember/object';
+import ethers from 'npm:ethers';
+//import formatKredits from 'kredits-web/utils/format-kredits.js';
```

Comment:
how do I do that import here? somehow I did not get this working.

Reply:
Without `.js` or a relative import: `import formatKredits from '../format-kredits';`

Reply:
ahhh, probably the only option that I did not try...thx.